tool/tctl/common/auth_command.go

```diff
@@ -144,6 +146,9 @@ func (a *AuthCommand) Initialize(app *kingpin.Application, config *service.Confi
 
 	a.authLS = auth.Command("ls", "List connected auth servers")
 	a.authLS.Flag("format", "Output format: 'yaml', 'json' or 'text'").Default(teleport.YAML).StringVar(&a.format)
+
+	a.authCRL = auth.Command("crl", "Export certificate revocation list (CRL) for certificates authorities")
```

Comment:
```suggestion
	a.authCRL = auth.Command("crl", "Export certificate revocation list (CRL) for certificate authorities")
```
We always generate an empty CRL, right? Should the help message indicate such?

Reply:
> We always generate an empty CRL, right? Should the help message indicate such?

Right, I'll update the message to indicate that.